dashboard-app/bower.json

```diff
@@ -34,6 +34,7 @@
     "ng-js-tree": "~0.0.7",
     "angular-ui-select": "^0.19.6",
     "angular-moment-picker": "~0.9.10"
```

Comment:
Comma (,) is missing at the end of this line. Bower install is failing because of this.